assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterServletModule.java

```diff
@@ -44,7 +44,11 @@ protected void configureServlets() {
             + "Origin,"
             + "Access-Control-Request-Method,"
             + "Access-Control-Request-Headers");
-    corsFilterParams.put("cors.support.credentials", "true");
+    if (Boolean.valueOf(System.getenv("CHE_CORS_ALLOW_CREDENTIALS"))) {
```

Comment:
I think `CHE_CORS_ALLOW__CREDENTIALS` would be better

Reply:
fixed